test/integration/sync_with_devto_test.rb

```
@@ -0,0 +1,176 @@
+    }
+    File.open("#{WORKING_DIR}/#{SYNC_STATUS_FILE}", 'w') { |file| file.write(articles.to_yaml) }
+  end
+
+  def test_sync_script_create_yaml_status_file_on_first_run
+    refute File.exist?("#{WORKING_DIR}/#{SYNC_STATUS_FILE}")
+    run_sync
+    assert File.exist?("#{WORKING_DIR}/#{SYNC_STATUS_FILE}")
+  end
+
+  def sync_file_content
+    YAML.load(File.read("#{WORKING_DIR}/#{SYNC_STATUS_FILE}"))
```

rubocop

[convention] Prefer using `YAML.safe_load` over `YAML.load`.

(Security/YAMLLoad)